### ISSUE TITLE
PiPifier does not work on some specific sites

### ISSUE THREAD

**Issue by jostrasser** [arnoappenzeller/PiPifier#93]
Hi,
I want to report that PiPifier stops working on some specific sites.
Here an example: 
https://tvthek.orf.at/live
https://tvthek.orf.at/schedule

Thanks!

**Comment by BrentMifsud** [comment]
I've noticed its not working on YouTube a lot of the time as well.

Tapping both the pip button on the video and the pip extension button does not seem to do anything.

Im on the latest version of safari and using an M1 Max MacBook Pro.

**Comment by stift-er** [comment]
> Have the same issue. A fix for me is to press CMD and drag the Icon in the Browser a few pixels and then release CMD. Works like originally intended for me.

**Comment by dirnb** [comment]
> I've noticed its not working on YouTube a lot of the time as well.
> 
> Tapping both the pip button on the video and the pip extension button does not seem to do anything.
> 
> Im on the latest version of safari and using an M1 Max MacBook Pro.

Have the same issue. A fix for me is to press CMD and drag the Icon in the Browser a few pixels and then release CMD. Works like originally intended for me.

**Comment by pflind** [comment]
Same here. I use it mainly on youtube and pip does not work in 9 of 10 times. Moving the icon didn't change anything.

**Comment by dirnb** [comment]
@pflind yeah it got worse for me as well. At some point it just didn't work what-so-ever. Switched over to [PiPer](https://github.com/amarcu5/PiPer), which works like a charm.

**Comment by maxengel** [comment]
I've found it no longer work with Sling TV.
